yt/frontends/arepo/fields.py

```diff
@@ -13,8 +13,7 @@
 
 class ArepoFieldInfo(GadgetFieldInfo):
     known_particle_fields = GadgetFieldInfo.known_particle_fields + \
-                            (("smoothing_length", ("code_length", [], None)),
```

Comment:
Is this backward compatibility breaking change ?

Reply:
This is a mistake, I'll fix it.

Reply:
Actually, no, it's not a mistake, because it's taken care of by the addition of this field to the `GadgetFieldInfo` class if it needs to exist.